tests/performance_tests/performance_test.py

```diff
@@ -54,6 +54,8 @@ def performPtbBinarySearch(tpsTestFloor: int, tpsTestCeiling: int, minStep: int,
     maxTpsAchieved = 0
     maxTpsReport = {}
     searchResults = []
+    if Utils.getNodeosVersion().split('.')[0] == "v2":
+        oldNodeos = True
```

Comment:
This check could be moved into `main()` and then `oldNodeos` could be set on `testClusterConfig`.